docs/deployment-guide-bicep.md

```diff
@@ -97,7 +97,10 @@ Parameter name | Default Value | Description
 -------------- | ------------- | -----------
 `deployDefender` | 'false' | When set to "true", enables Microsoft Defender for Cloud for the subscriptions used in the deployment. It defaults to "false".
 `emailSecurityContact` | '' | Email address of the contact, in the form of john@doe.com
+`pricingTier` | 'Standard' | The standard pricing tier offers advanced security capabilities
 
+The `pricingTier` for Microsoft Defender for Cloud is enabled by default in the following [Azure Environments](https://docs.microsoft.com/en-us/powershell/module/servicemanagement/azure.service/get-azureenvironment?view=azuresmps-4.0.0): `AzureCloud` and `AzureUSGovernment`. To enable this for other Azure Cloud environments, this will need to executed manually. Documentation on how to do this is found
```

Comment:
The Defender user interface calls this "Defender plan by resource type" rather than `pricingTier`, so would it be better to describe this in terms of what the portal calls it instead of the underlying resource provider? That would tell people what to look for when manually changing the setting in the portal.

Reply:
Updated in commit [4c7a06c](https://github.com/Azure/missionlz/pull/687/commits/4c7a06c301a8d9b9e54b2e72a4a58644c3607fbb)